src/server/req_quejob.c

```diff
@@ -3236,7 +3236,7 @@ long long get_next_svr_sequence_id(void)
 	++svr_sequence_window_count;
 	ret_svr_sequence_id = svr_jobidnumber;
 	/* sequence window count is more than 1000, reset back to zero*/
-	if (svr_sequence_window_count > SEQ_WIN_INCR) {
```

Comment:
As per my understanding, we can't put this line in the end because at line 3243 we are checking the next jobid that should not be greater than the svr_max_job_sequence_id attribute. If  greater then need to wrap it to 0 from here itself.
Will not loose the jobid because we are assigning first (at line 3223) and then increasing the jobid later.
I think previously also PBS saves the advance jobid in the database and written a testcase as well to check this scenario.

Reply:
Ok, in that case change the operator in te check to be '>=' just to be on a safer side

Reply:
Done!